cypress/e2e/hub/approvals.cy.ts

```diff
@@ -12,6 +12,7 @@ describe('Approvals', () => {
     cy.hubLogin();
```

Comment:
@acruzgon Please review the approval tests changes.

Reply:
all tests passed
![image](https://github.com/ansible/ansible-ui/assets/88343229/1b4f5e2c-cd01-48ec-9f8f-58f174da4ace)